### DISCUSSION TITLE
Difference between `mergeMap` and `flatMap`?

### DISCUSSION THREAD

**Discussion by Sawtaytoes** [ReactiveX/rxjs#6075] [Q&A] [answered]
It was my understanding `flatMap` was the old name for `mergeMap`.

Looking at the docs, there are things about `mergeMap`, but nothing denoting `flatMap` as a deprecated alias: https://rxjs.dev/api/operators/flatMap.

Am I wrong to think these are the same thing?

**Comment by cartant** [comment, marked as answer]
https://github.com/ReactiveX/rxjs/blob/c6ba9790ed42e4781278822aff96bce0863e7917/src/internal/operators/mergeMap.ts#L99
